src/index.js

```diff
@@ -0,0 +1,636 @@
+  {
+    shopifyField: "cartLine.merchandise.image.src",
+    rudderField: "image_url",
+  },
+  {
+    shopifyField: "cartLine.merchandise.price.amount",
+    rudderField: "price",
+  },
+  {
+    shopifyField: "cartLine.merchandise.product.id",
+    rudderField: "product_id",
+  },
+  {
+    shopifyField: "cartLine.merchandise.product.title",
+    rudderField: "variant",
+  },
+  {
+    shopifyField: "cartLine.merchandise.product.type",
+    rudderField: "category",
+  },
+  {
+    shopifyField: "cartLine.merchandise.product.vendor",
+    rudderField: "brand",
+  },
+  {
+    shopifyField: "cartLine.merchandise.product.url",
+    rudderField: "url",
+  },
+  {
+    shopifyField: "cartLine.merchandise.sku",
+    rudderField: "sku",
+  },
+  {
+    shopifyField: "cartLine.merchandise.title",
+    rudderField: "name",
+  },
+  {
+    shopifyField: "cartLine.quantity",
+    rudderField: "quantity",
+  },
+];
+
+const cartViewedEventMapping = [
+  { shopifyField: "merchandise.product.id", rudderField: "product_id" },
+  { shopifyField: "merchandise.product.title", rudderField: "variant" },
+  { shopifyField: "merchandise.image.src", rudderField: "image_url" },
+  { shopifyField: "merchandise.price.amount", rudderField: "price" },
+  { shopifyField: "merchandise.product.type", rudderField: "category" },
+  { shopifyField: "merchandise.product.url", rudderField: "url" },
+  { shopifyField: "merchandise.product.vendor", rudderField: "brand" },
+  { shopifyField: "merchandise.sku", rudderField: "sku" },
+  { shopifyField: "merchandise.title", rudderField: "name" },
+  { shopifyField: "quantity", rudderField: "quantity" },
+];
+
+const productListViewedEventMapping = [
+  { shopifyField: "image.src", rudderField: "image_url" },
+  { shopifyField: "price.amount", rudderField: "price" },
+  { shopifyField: "product.id", rudderField: "product_id" },
+  { shopifyField: "product.title", rudderField: "variant" },
+  { shopifyField: "product.type", rudderField: "category" },
+  { shopifyField: "product.url", rudderField: "url" },
+  { shopifyField: "product.vendor", rudderField: "brand" },
+  { shopifyField: "sku", rudderField: "sku" },
+  { shopifyField: "title", rudderField: "name" },
+];
+
+const checkoutStartedCompletedEventMapping = [
+  { shopifyField: "quantity", rudderField: "quantity" },
+  { shopifyField: "title", rudderField: "name" },
+  { shopifyField: "variant.image.src", rudderField: "image_url" },
+  { shopifyField: "variant.price.amount", rudderField: "price" },
+  { shopifyField: "variant.sku", rudderField: "sku" },
+  { shopifyField: "variant.product.id", rudderField: "product_id" },
+  { shopifyField: "variant.product.title", rudderField: "variant" },
+  { shopifyField: "variant.product.type", rudderField: "category" },
+  { shopifyField: "variant.product.url", rudderField: "url" },
+  { shopifyField: "variant.product.vendor", rudderField: "brand" },
+];
+
+(function () {
+  "use strict";
+  window.RudderSnippetVersion = "3.0.14";
```

Comment:
The snippet is a bit old. Can we use the latest one from the dashboard?

Reply:
@yashasvibajpai can you get latest snippet from source setup page in webapp & test if everything is intact ?

Reply:
updated